.github/workflows/test-coverage.yml

```diff
@@ -0,0 +1,31 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+
+name: test-coverage
+
+jobs:
+  test-coverage:
+    runs-on: macos-latest
```

Comment:
Can you try once with `runs-on: windows-latest`?

Reply:
Daniel uses Windows machine, so whatever skipping logic is written, it is around that OS. So maybe there are better chances of seeing which tests fail.

Or are you also using Windows?

Reply:
I'm also on windows and it works, let's see if it works with windows-latest